src/client/search/SearchTermsReducer.js

```diff
@@ -0,0 +1,27 @@
+const SearchTermsReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'ADD_SEARCH_TERM':
+      return [
+      ...state,
+      SearchTermReducer(undefined, action),
+			];
+    default:
```

Comment:
I don't understandwhere do you want curly braces ? 

you shouldn't be debugging a reducer from here anyway... you should either have tests or be using the redux debug tool